src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,222 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+    private Node<T> node;
```

Comment:
we use this as temporary variable. Do we need tho make field from this?

Reply:
done